### PR TITLE
Fix some teething problems

### LINKED ISSUE #7: Support new Cylc 8 filenames
flow.cylc - https://github.com/cylc/cylc-admin/blob/16b52f057c944d1fb15d81ed6827050d6617363b/docs/proposal-config-changes.md#config-file-naming

### PR DESCRIPTION
**Pull request by MetRonnie**
This will also resolve #7

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
                 ],
                 "filenamePatterns": [
                     "suite*.rc",
-                    "suite*.rc.*"
+                    "suite*.rc.*",
+                    "*.cylc"
```
Comment (MetRonnie) [inline package.json:30]: Is this okay? Or do we not want the same grammar for `global.cylc` as for `flow.cylc`, in which case it should be changed to `flow*.cylc`?

Comment (oliver-sanders) [reply package.json:30]: Fine, there will be other .cylc files.

**Review by oliver-sanders — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (datamel) [inline package.json:30]: Just a thought, would adding global.rc, flow.rc, flow-tests.rc make sense? I have loaded my files of these, renaming them to suite.rc and the extension works well?

**Review by datamel — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (oliver-sanders) [reply package.json:30]: We could make the change to `flow.cylc` pretty soon so I think this'll do for now.

**Pull request merged**